allowContentType(s) expression docs #316 (#317)
enonic/doc-xp: docs/cms/content-types.adoc

<8> *allow-child-content* (optional; default: true) If false, no content will be allowed to be created or moved under content of this content type
(e.g. prevents child content under media)
<9> *allow-child-content-type* (optional) If specified, only content of content types matching specified criteria will be allowed

> to be created or moved under content of this content type. Only <<input-types#allowContentType, MATCH>> pattern mode is available. image:xp-770.svg[XP 7.7.0,opts=inline]

Reviewer: ```suggestion
to be created or moved under content of this content type. Pattern matching is the same as described in <<input-types#allowContentType, MATCH>>. image:xp-770.svg[XP 7.7.0,opts=inline]
```
